<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>386</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="MessagesWindow">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MessageTextTextEdit" name="MessageText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SendEmoji">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>App Resources/SendEmojiButton.png</normaloff>App Resources/SendEmojiButton.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SendFile">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>App Resources/SendAttatchmentButton.png</normaloff>App Resources/SendAttatchmentButton.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SendMessage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>App Resources/SendMessageButton.png</normaloff>App Resources/SendMessageButton.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MessageTextTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">MessageTextTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
